src/Components/External-Navbar/NavbarItem/component.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react';
+import Links from '../../Link/component';
+import Nav from 'react-bootstrap/Nav';
+
+type ItemProps = {
```

Comment:
All props that all not used in this exact file are not need to be mentioned here.